src/AppBundle/Service/SearchUserFormHelper.php

```diff
@@ -22,11 +22,13 @@ class SearchUserFormHelper
 {
     private $container;
     private $use_fly_and_fixed;
+    private $maximum_nb_of_beneficiaries_in_membership;
 
-    public function __construct(ContainerInterface $container, $use_fly_and_fixed)
+    public function __construct(ContainerInterface $container)
```

Comment:
@samueleyre est-ce que c'est une bonne pratique de récupérer les paramètres via le container plutôt que via le services.yml + construct ? Ca clean le code en tout cas je trouve !

Reply:
De ce que j'ai lu sur le sujet, ce n'est pas trop recommandé pour cette version de symfony, mais je trouve aussi que c'est plus clair. 

En symfony 4.1, on peut utiliser le "[ParameterBagInterface](https://symfony.com/blog/new-in-symfony-4-1-getting-container-parameters-as-a-service)" pour avoir accès aux paramètres donc je pense que c'est pas une mauvaise chose de ne pas avoir à spécifier tous les paramètres utiles dans service.yml.